app/assets/v2/css/rain.css

```diff
@@ -1,763 +1,155 @@
 .rain {
+  position: fixed;
+  top: -50%;
   background: white;
   height: 100px;
-  position: absolute;
   width: 6px;
-  transform: rotate(40deg);
-  border-radius: 5px;
+  border-radius: 15px;
+  z-index: 0;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+  cursor: default;
+  will-change: transform, top, left;
+  -webkit-animation-name: rains-fall, rains-shake;
+          animation-name: rains-fall, rains-shake;
+  -webkit-animation-duration: 20s, 8s;
+          animation-duration: 20s, 8s;
+  -webkit-animation-timing-function: linear, linear;
+          animation-timing-function: linear, linear;
+  -webkit-animation-iteration-count: infinite, infinite;
+          animation-iteration-count: infinite, infinite;
+  -webkit-animation-play-state: running, running;
+          animation-play-state: running, running;
 }
 
-/*
-.rain.color:nth-of-type(3n+0) {
-	background: linear-gradient(to bottom, rgba(255, 255, 255, 0) 0%, #0d0d6d 100%);
-}
-.rain.color:nth-of-type(3n+1) {
-	background: linear-gradient(to bottom, rgba(255, 255, 255, 0) 0%, #f41f7a 100%);
-}
-.rain.color:nth-of-type(3n+2) {
-	background: linear-gradient(to bottom, rgba(255, 255, 255, 0) 0%, #4aeba8 100%);
-}
-.rain.grey {
-	background: linear-gradient(to bottom, rgba(255, 255, 255, 0) 0%, #f1e1e1 100%);
-}
-*/
-
-.rain.color:nth-of-type(3n+0) {
-  background: #0d0d6d;
-}
-
-.rain.color:nth-of-type(3n+1) {
-  background: #f41f7a;
-}
-
-.rain.color:nth-of-type(3n+2) {
-  background: #4aeba8
+.rain:nth-of-type(3n+0) {
+  background: rgba(13, 13, 109, 0.4);
 }
 
-.rain.grey {
-  background: #f1e1e1;
+.rain:nth-of-type(3n+1) {
+  background: rgba(244, 31, 122, 0.4);
 }
 
-.rain.purple {
-  background-color: var(--gc-purple);
+.rain:nth-of-type(3n+2) {
```

Comment:
no more `rain.grey`?

Reply:
looks like `rain.purple` is also used

Reply:
That classes wasn't being used. I think was old code since then was changed to nth-of-type to determine colors